### DISCUSSION TITLE
Is there any way to specify the destination path of predicted labels?

### DISCUSSION THREAD

**Discussion by dkobayas-cyber** [ultralytics/ultralytics#2036] [Q&A] [answered]
Hello, 

When we predict labels by: 
```python 

model = YOLO("path/to/best/weight.pt")
result = model.predict(source="path/to/test/images/", save_txt=True)

```
the predictor creates a directory named `predict[1-9]/` in `runs` directory to save the results by default. 

I'd like to define the destination of the results somehow. Is there anyway to specify that? I checked `ultralytics/yolo/cfg/__init__.py` but I couldn't find corresponding configurations.

**Comment by robmarkcole** [comment]
Bumping this up - I want to specify where the `runs/` folder is created

**Reply by pathfinder2002** [reply]
also interested.

**Reply by glenn-jocher** [reply, marked as answer]
@pathfinder2002 hello! 👋

When using the `predict` mode in YOLOv8, the results are indeed saved in a `runs/predict` directory by default. To specify a different destination path for the predicted labels, you can use the `project` and `name` arguments when calling the `predict` method. The `project` argument allows you to set the base directory, while the `name` argument lets you specify the name of the run directory within the base directory.

Here's how you can use these arguments:

```python
result = model.predict(source="path/to/test/images/", save_txt=True, project="path/to/destination", name="custom_run_name")
```

This will save the results in `path/to/destination/custom_run_name/`.

I hope this helps! If you have any more questions, feel free to ask. Happy detecting! 😊
